cardano-sl.cabal

```diff
@@ -120,6 +120,7 @@ library
                         Pos.Util.Arbitrary
                         Pos.Util.BackupPhrase
                         Pos.Util.JsonLog
+                        Pos.Util.Servant
```

Comment:
@akegalj wants it to be used in explorer some day as well.
Does it worth separate condition `if flag(with-web) || flag (with-explorer)`?